owncloudApp/src/main/res/drawable/ic_folder.xml

```diff
@@ -1,6 +1,6 @@
 <vector xmlns:android="http://schemas.android.com/apk/res/android"
-        android:width="24dp"
-        android:height="24dp"
+        android:width="48dp"
+        android:height="48dp"
```

Comment:
If we need a bigger icon, we can set the size where we need to use it. This is a vector that will scale properly to that size instead of increasing its base size. Is this right?

Reply:
Yep, that's right, but it is analogue to `ic_spaces` (it is used in the bottom bar and in the spaces cards) and it is easier to set a bigger size in the original XML since it won't change the appearance of any of its previous uses